src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -66,7 +71,7 @@ public PersonBuilder withName(String name) {
      * Parses the {@code clientSources} into a {@code Set<ClientSource>}
      * and set it to the {@code Person} that we are building.
      */
-    public PersonBuilder withClientSources(String ... clientSources) {
+    public PersonBuilder withClientSources(String... clientSources) {
```

Comment:
Perhaps we should add back the space here

Reply:
uhm this string matters? But ok changed.